backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -16,6 +17,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Name, src => src.Name)
                 .Map(dest => dest.ReportedFiscalYear, src => src.ReportedFiscalYear)
                 .Map(dest => dest.ActualFiscalYear, src => src.ActualFiscalYear)
+                .Map(dest => dest.NetBookFiscalYear, src => src.NetBookFiscalYear)
```

Comment:
I don't think we need this.  The NetBook value for the Project will be the ActualFiscalYear.

Reply:
Agreed, I think this was from the v1 of the design.